packages/calcite-components/src/components/tab/tab.scss

```diff
@@ -13,6 +13,10 @@
   @apply block h-full w-full overflow-auto;
 }
 
+.content {
+  @apply box-border;
```

Comment:
Sidebar: @driskull Do you think we could split up https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/assets/styles/_host.scss for supporting/child components if they don't need background or other styles? Or if all components should be using `%component-host`, we can set up a rule to enforce it across all component style files.